File: src/main/java/com/codingspezis/android/metalonly/player/wish/WishPrefs.java
package com.codingspezis.android.metalonly.player.wish;

import org.androidannotations.annotations.sharedpreferences.DefaultString;
import org.androidannotations.annotations.sharedpreferences.SharedPref;

@SharedPref
public interface WishPrefs {
    @DefaultString("")
    String nick();
    @DefaultString("")
    String artist();
    @DefaultString("")
    String title();
    @DefaultString("")
    String greeting();
}

File: src/main/kotlin/com/codingspezis/android/metalonly/player/wish/WishPrefs.kt
package com.codingspezis.android.metalonly.player.wish

import org.androidannotations.annotations.sharedpreferences.DefaultString
import org.androidannotations.annotations.sharedpreferences.SharedPref

@SharedPref
interface WishPrefs {
    @DefaultString("")
    fun nick(): String

    @DefaultString("")
    fun artist(): String

    @DefaultString("")
    fun title(): String

    @DefaultString("")
    fun greeting(): String
}
